docs/docs/tutorials/helidon-integration.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebar_position: 29
+---
+
+# Helidon Integration
+
+[Helidon](https://helidon.io/) provides a LangChain4j integration module that simplifies to build AI-driven applications while leveraging Helidon’s programming model and style.
+
+You can find the detailed explanation and usage of LangChain4j integration feature [here](https://helidon.io/docs/latest/se/integrations/langchain4j/langchain4j).
+
+## Supported versions
+
+Helidon's LangChain4j integration requires Java 21 and Helidon 4.2.
+
+## Examples
+
+We have created several sample applications for you to explore. These samples demonstrate all aspects of using LangChain4J in Helidon applications.
```

Comment:
"samples demonstrate how to use LangChain4j in Helidon applications". 

Please note 'j' in LangChain4j is in lower case.

Reply:
Yes, that's how its used in official docs https://docs.langchain4j.dev/ too i.e. lower case j